README.md

```diff
@@ -15,28 +46,44 @@ npm i @po.et/poet-js
 
 ## Usage
 
-The main function you'll be using is `createClaim`:
+First create a `ClaimSigner` to sign and verify your claims. Note that the Po.et network currently uses 
+[Ed25519Signature2018](https://w3c-dvcg.github.io/lds-ed25519-2018/), which requires a Base58
+form of the Ed25519 Private Key. You can use the KeyHelper utility to generate a base58 public/privateKey pair, if you
+do not yet have one.
+
+```typescript
+import { ClaimSigner, KeyHelper } from '@po.et/poet-js'
+
+const { privateKey } = KeyHelper.generateED25519Base58Keys('password') // e.g 'LWgo1jraJrCB2QT64UVgRemepsNopBF3eJaYMPYVTxpEoFx7sSzCb1QysHeJkH2fnGFgHirgVR35Hz5A1PpXuH6'
+
+const claimSigner = ClaimSigner(privateKey)
```

Comment:
Will this also take a publicKey for RSA in the future?

Reply:
Once I find a decent (small) generatePublicKeyFromPrivateKey implementation for RSA

Reply:
I am unsure what you mean. You are still looking for a private to public key function, so your hoping ClaimSigner will not need to take a public key?

Reply:
Correct.

Reply:
Ok cool 👍

Reply:
This is much nicer than creating the object manually!

Reply:
👍